PreSheaves/gap/PreSheaves.gi

```diff
@@ -1058,6 +1050,14 @@ InstallMethod( YonedaEmbeddingOfSourceCategory,
     
 end );
 
+##
+InstallOtherMethod( \.,
+        "for a presheaf and a positive integer",
+        [ IsPreSheafCategory, IsPosInt ],
+  
+  { PSh, string_as_int } -> ApplyFunctor( YonedaEmbeddingOfSourceCategory( PSh ), Source( PSh ).( NameRNam( string_as_int ) ) )
```

Comment:
Why is this code not covered?

Reply:
Actually, no serious computation in the presheaves category is needed to decide the congruence of morphisms in the quotient category of an algebroid. However, I have updated the PR to include some tests to presheaves.